.changeset/red-lamps-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(InfoGroup): add InfoGroup component
```

Comment:
why is the InfoGroup component here?

Reply:
removed it. I had created this stacked on top of infogroup